src/Catty/Views/Custom/CollectionViewCells/BrickCells/Look/SetBackgroundAndWaitBrickCell.swift

```diff
@@ -0,0 +1,53 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
```

Comment:
Please change to  *  Copyright (C) 2010-2021 The Catrobat Team

Reply:
Done.